python/metashape_workflow.py

```diff
@@ -78,3 +78,5 @@ def parse_args():
 
 ### Run the Metashape workflow
 meta.run()
+
+meta.dump_paths_to_json()
```

Comment:
A comment could be helpful. Also, this function returns the filename that the json data is written to. Would it be helpful to print this to standard out in a way that can be consumed by later workflow steps? Alternatively, it sounds like you may change this to directly printing json representation of the paths, instead of saving to the file.